src/NServiceBus.AcceptanceTests/Core/Outbox/When_outbox_enabled_on_send_only.cs

```diff
@@ -0,0 +1,83 @@
+﻿namespace NServiceBus.AcceptanceTests.Core.Outbox
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using Features;
+    using Microsoft.Extensions.DependencyInjection;
+    using NServiceBus.Configuration.AdvancedExtensibility;
+    using NServiceBus.Outbox;
+    using NServiceBus.Persistence;
+    using NUnit.Framework;
+
+    public class When_outbox_enabled_on_send_only : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Should_make_outbox_and_storage_session_available()
+        {
+            var context = await Scenario.Define<Context>()
+                .WithEndpoint<Endpoint>()
+                .Done(c => c.EndpointsStarted)
+                .Run();
+
+            Assert.True(context.HasOutboxStorage);
+            Assert.True(context.HasSynchronizedStorageSession);
+        }
+
+        class Context : ScenarioContext
+        {
+            public bool HasOutboxStorage { get; set; }
+            public bool HasSynchronizedStorageSession { get; set; }
+        }
+
+        class Endpoint : EndpointConfigurationBuilder
+        {
+            public Endpoint()
+            {
+                EndpointSetup<DefaultServer>(c =>
+                {
+                    c.GetSettings().Set("Outbox.Enabled", true);
```

Comment:
Not the best key, should we use something more explicit? (Outbox.EnableOnSendOnly ?)

Reply:
I think a more explicit key could be good. On the other hand, you could also name it something like `Outbox.DisablePrerequisiteChecks`

Reply:
I like it, will change